src/core/public/http/types.ts

```diff
@@ -92,6 +97,21 @@ export interface IBasePath {
   remove: (url: string) => string;
 }
 
+/**
+ * APIs for denoting paths as not requiring authentication
+ */
+export interface IAnonymousPaths {
+  /**
+   * Determines whether the provided path doesn't require authentication
```

Comment:
Since this is a public service, we should explain that the `basePath` will be automatically accounted for if included as part of the `path`.